plumpy/process_comms.py

```diff
@@ -195,10 +217,15 @@ async def kill_process(self, pid, msg=None):
         future = await asyncio.wrap_future(kill_future)
         # Now wait for the kill to be enacted
         result = await asyncio.wrap_future(future)
-
         return result
 
-    async def continue_process(self, pid, tag=None, nowait=False, no_reply=False):
+    async def continue_process(
+        self,
+        pid: 'PID_TYPE',
+        tag: Optional[str] = None,
+        nowait: bool = False,
+        no_reply: bool = False
+    ) -> 'ProcessResult':
```

Comment:
if `no_reply=True` method return `None`. So I guess here we have `->Union[None, 'ProcessResult']`

Reply:
```suggestion
    ) -> Optional['ProcessResult']:
```